downgrade

```diff
@@ -38,6 +38,10 @@ download_when_remote() {
       rm "$pkg"
       return 1
     fi
+    if ! curl --progress-bar "${url}.sig" > "${pkg}.sig"; then
```

Comment:
Does just downloading the signature file cause it to be checked?

Are we assuming here that there will always be a `.sig` available?